apps/aecore/lib/aecore/contract/tx/contract_call_tx.ex

```diff
@@ -208,7 +208,7 @@ defmodule Aecore.Contract.Tx.ContractCallTx do
 
   @spec preprocess_check(
           Chainstate.accounts(),
-          tx_type_state(),
+          Chainstate.t(),
```

Comment:
this is still not the correct spec, as the behaviour callbacks state `tx_type_state()`